input/blog/2020-12-20-cake-v1.0.0-rc0002-released.md

```diff
@@ -0,0 +1,104 @@
+﻿---
+title: Cake v1.0.0-rc0002 released
+category: Release Notes
+author: devlead
+---
+
+Version 1.0.0-rc0002 of Cake has been released.
+
+This is the second, and probably last, pre-release version of Cake 1.0. Focus in this release has been to bring Cake Frosting up to speed, align it with the Cake script runners and bug fixes.
+
+### Major new features
+
+* Frosting NuGet tool resolution
+* Align Cake Frosting CLI with Cake script runners
+* Bug fixes
+* Addin recommended Cake version set to 1.0
```

Comment:
Should we explain this a little bit more, since this means that there now will be a warning shown for all addins, until 1.0 will be out and addins updated.

Reply:
I'll try to add a blurb.

Reply:
Added an `Addin recommended Cake version set to 1.0` section,